packages/common/src/ui/layout/tables/columns/NameAndColorColumn.tsx

```diff
@@ -37,11 +37,11 @@ export const getNameAndColorColumn = <
       }}
     >
       <ColorSelectButton
-        onChange={color => column.setter({ ...rowData, color })}
+        onChange={color => column.setter({ ...rowData, color: color.hex })}
         color={rowData.color}
       />
       <Box ml={1} />
-      {column.accessor(rowData)}
+      {column.accessor({ rowData, rows })}
```

Comment:
I suppose the downside of leaving it in is that whenever you call it you need the rows 🤔